tests/test_special_ops.py

```diff
@@ -1189,6 +1190,7 @@ def test_accuracy_kron(shape, dtype):
     gems_assert_equal(res_out, ref_out)
 
 
+@pytest.mark.skipif(flag_gems.device == "musa", reason="temp skip")
```

Comment:
Please confirm the skip condition and ensure its consistency across all test cases.